### ISSUE TITLE
reduce output

### ISSUE THREAD

**Issue by alexcb** [earthly/earthly#141]
the progress statuses can take up a lot of output; perhaps we can change to an ncurses-like progress bar (similar to buildkit's), or change the logic to only print out values when they changes (either by 1, or 5 percentage points, or even once a second).

here's the code:
https://github.com/earthly/earthly/blob/master/builder/solver.go#L298

and here's an example of too much output:

```
./examples/tests+from-expose-test | +base./examples/tests+from-expose-test |  | ./examples/tests+from-expose-test | sha256:df20fa9351a15782c64e6dddb2d4a6f50bf6d3688060a34c4014b0d9a752eb4c 44%
./examples/tests+from-expose-test | +base./examples/tests+from-expose-test |  | ./examples/tests+from-expose-test | sha256:df20fa9351a15782c64e6dddb2d4a6f50bf6d3688060a34c4014b0d9a752eb4c 47%
./examples/tests+git-clone-test | +base | sha256:d838eb6031fe0bb4f1f203ec26c293afccce4db86abde5d3b1c3a923cdca5120 32%
./examples/tests+git-clone-test | +base | sha256:d838eb6031fe0bb4f1f203ec26c293afccce4db86abde5d3b1c3a923cdca5120 33%
./examples/tests+excludes-test | +base | sha256:cbdbe7a5bc2a134ca8ec91be58565ec07d037386d1f1d8385412d224deafca08 0%
./examples/tests+excludes-test | +base./examples/tests+excludes-test |  | sha256:cbdbe7a5bc2a134ca8ec91be58565ec07d037386d1f1d8385412d224deafca08 0%
./examples/tests+excludes-test | +base./examples/tests+excludes-test |  | ./examples/tests+excludes-test | sha256:cbdbe7a5bc2a134ca8ec91be58565ec07d037386d1f1d8385412d224deafca08 0%
./examples/tests+git-clone-test | +base./examples/tests+git-clone-test |  | ./examples/tests+git-clone-test | sha256:d838eb6031fe0bb4f1f203ec26c293afccce4db86abde5d3b1c3a923cdca5120 34%
./examples/tests+git-clone-test | +base./examples/tests+excludes-test | +base./examples/tests+excludes-test |  | ./examples/tests+excludes-test | sha256:cbdbe7a5bc2a134ca8ec91be58565ec07d037386d1f1d8385412d224deafca08 0%
./examples/tests+excludes-test | +base./examples/tests+excludes-test |  | sha256:cbdbe7a5bc2a134ca8ec91be58565ec07d037386d1f1d8385412d224deafca08 0%
```

**Comment by abennett** [comment]
Is this resolved since #143?

**Comment by alexcb** [comment]
Good eyes @abennett, it's a duplicate of #84, and has in fact been fixed.
